src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.searchBar {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(225, 231, 237, 1);
+}
+
+.searchBar__small {
+  height: 42px;
+  padding-left: 33px;
+
+  font-size: 14px;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar::placeholder {
+  font-family: "Avenir", sans-serif;
```

Comment:
you can use here `inherit` you already added font-family style

Reply:
Should I use 'inherit' for font-size and font-weight in all other cases like in .searchBar and .searchBar::placeholder?